### PR TITLE
Create Azure Purview workbook

### PR DESCRIPTION
**Pull request by libarson**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: Workbooks/Azure Purview [hunk 1]
```diff
@@ -0,0 +1,1497 @@
...
+            "style": "link"
+          }
+        ]
+      },
+      "name": "links - 9"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "__Azure Purview__\r\n\r\nAzure Purview is a unified data governance service that helps you manage and govern your on-premises, multicloud, and software-as-a-service (SaaS) data. It creates a holistic, up-to-date map of your data landscape with automated data discovery, sensitive data classification, and end-to-end data lineage that empowers you to find valuable and trustworthy data.  <a href='https://azure.microsoft.com/services/purview/' target='_blank'>Learn More</a> \r\n\r\n",
```
Comment (saasri) [inline Workbooks/Azure Purview:163]: 'on-prem'

File: Workbooks/Azure Purview [hunk 2]
```diff
@@ -0,0 +1,1497 @@
...
...fication, ClassificationCount, SensitivityLabelName;\r\n\r\nlet labeledAssets = PurviewDataSensitivityLogs\r\n| where ActivityType == \"Labeling\" \r\n| where \"{PurviewAccount:label}\" == \"All\" or PurviewAccountName in~ (split(\"{PurviewAccount:label}\", \", \"))\r\n| where \"{DataSource:label}\" == \"All\" or SourceType in~ (split(\"{DataSource:label}\", \", \"))\r\n| where SensitivityLabelName != int(null)\r\n| extend newSensitivityLabelName = iif(isempty(SensitivityLabelName), \"No Label\", SensitivityLabelName)\r\n| summarize arg_max(newSensitivityLabelName, SourceType) by AssetPath\r\n| project AssetPath, newSensitivityLabelName;\r\n\r\nlet table = classifiedAssets\r\n| join kind= leftouter labeledAssets on AssetPath\r\n| project AssetPath, newSensitivityLabelName, ClassificationCount, Classification, TimeGenerated, PurviewTenantId, PurviewSubscriptionId, PurviewAccountName, PurviewRegion, SourceName, SourceType, SourcePath, SourceSubscriptionId, SourceRegion, SourceCollectionName, AssetName, AssetType, AssetModifiedTime, AssetLastScanTime, FileExtension, FileSize, ActivityType, ActivityTrigger\r\n| sort by ClassificationCount;\r\n\r\ntable\r\n\r\n\r\n\r\n\r\n\r\n\r\n\r\n",
```
Comment (saasri) [inline Workbooks/Azure Purview:367]: 'newSensitivityLabelName' looks off, just 'LabelName' works, you should try to keep this consistent across queries. 'AssetCreationTime' is missing

**Comment by v-rucdu** [comment]
@libarson : For deploying any workbook its entry needs to be added in [WorkbooksMetadata.json](https://github.com/Azure/Azure-Sentinel/blob/master/Workbooks/WorkbooksMetadata.json).

Please refer [instructions](https://github.com/Azure/Azure-Sentinel/blob/master/Workbooks/README.md) to understand more on how to add a new workbook.

File: Workbooks/AzurePurview.json [hunk 3]
```diff
@@ -0,0 +1,1452 @@
...
+        "value": "Labels"
+      },
+      "customWidth": "50",
+      "name": "query - 13",
+      "styleSettings": {
+        "showBorder": true
+      }
+    }
+  ],
+  "fallbackResourceIds": [
```
Comment (v-rucdu) [inline Workbooks/AzurePurview.json:1448]: Please remove the reference to Azure resource by making the fallbackResourceIds attribute empty as below.
"fallbackResourceIds": [],

Thanks,
Ruchita

**Review by v-rucdu — CHANGES_REQUESTED** [review]

**Comment by v-rucdu** [comment]
Please provide atleast 2 images one for white theme and other for black theme.

**Comment by libarson** [comment]
@v-rucdu I added in the black/white images along with entry to the WorkbooksMetadata.json file. I also removed the reference in fallbackResourceIds. 

I see that some of the checks are failing, can you let me know what I need to update to allow them to pass?

File: Workbooks/WorkbooksMetadata.json [hunk 4]
```diff
@@ -1415,5 +1415,18 @@
...
       "subtitle": "",
       "provider": "Azure Sentinel Community"
+    },   
+    {
+      "workbookKey": "AzurePurview",
+      "logoFileName": "AzurePurview.svg",
+      "description": "This workbook provides insights into the sensitivity of data that has been scanned by Azure Purview, including classifications and labels found across the data estate.",
+      "dataTypesDependencies": ["PurviewDataSensitivityLogs"],
+      "dataConnectorsDependencies": ["AzurePurview"],
+      "previewImagesFileNames": ["AzurePurviewWhite1.png", "AzurePurviewBlack1.png", "AzurePurviewWhite2.png", "AzurePurviewBlack2.png","AzurePurviewWhite3.png", "AzurePurviewBlack3.png"],
```
Comment (v-rucdu) [inline Workbooks/WorkbooksMetadata.json:1425]: Please maintain consistency in the file extensions, while creating and mentioning it in Metadata file. This is one of the reason for validation failure.

e.g. File is added with name as  **AzurePurviewWhite1.PNG** under preview folder and is being referenced as **AzurePurviewWhite1.png** in metadata file. 

Please fix it for all the images.

Thanks!!!

**Review by v-rucdu — CHANGES_REQUESTED** [review]

File: Workbooks/Images/Logos/AzurePurview.svg [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+<path d="M17.73 8.379C16.7528 7.20844 15.6242 6.17303 14.374 5.3C13.0667 4.94302 11.6867 4.94993 10.383 5.32C9.99125 5.43923 9.61342 5.60015 9.256 5.8C9.81451 5.83934 10.353 6.02413 10.818 6.336C10.8597 6.34793 10.8994 6.36578 10.936 6.389C11.7055 6.90574 12.3597 7.57626 12.8572 8.35832C13.3548 9.14038 13.685 10.017 13.827 10.933C13.9186 11.6512 13.9075 12.3789 13.794 13.094C15.2758 12.1523 16.6023 10.986 17.726 9.637C17.8123 9.55665 17.8812 9.45945 17.9284 9.3514C17.9757 9.24336 18.0002 9.12678 18.0006 9.00887C18.001 8.89095 17.9771 8.77422 17.9306 8.66588C17.8841 8.55754 17.8158 8.4599 17.73 8.379V8.379Z" fill="url(#paint3_linear)"/>
+<path d="M21.379 -3.738H-2.9V24.033H21.379V-3.738Z" stroke="#B31B1B" stroke-miterlimit="10"/>
+</g>
+<defs>
+<linearGradient id="paint0_linear" x1="8.977" y1="11.134" x2="8.977" y2="7.178" gradientUnits="userSpaceOnUse">
+<stop stop-color="#29BADE"/>
+<stop offset="1" stop-color="#50E6FF"/>
+</linearGradient>
+<linearGradient id="paint1_linear" x1="6.446" y1="14.626" x2="6.446" y2="3.374" gradientUnits="userSpaceOnUse">
```
[the remark this answers is not in the document]
Comment (libarson) [reply Workbooks/Images/Logos/AzurePurview.svg]: Updated the svg IDs to unique GUIDs.

File: Workbooks/WorkbooksMetadata.json (hunk 4 above)
Comment (libarson) [reply Workbooks/WorkbooksMetadata.json]: Fixed the file extensions for the images in the preview folder.

File: Workbooks/WorkbooksMetadata.json [hunk 6]
```diff
@@ -1415,5 +1415,18 @@
       "templateRelativePath": "AzureSentinelCost.json",
       "subtitle": "",
       "provider": "Azure Sentinel Community"
+    },   
+    {
+      "workbookKey": "AzurePurview",
+      "logoFileName": "AzurePurview.svg",
+      "description": "This workbook provides insights into the sensitivity of data that has been scanned by Azure Purview, including classifications and labels found across the data estate.",
+      "dataTypesDependencies": ["PurviewDataSensitivityLogs"],
```
Comment (v-rucdu) [inline Workbooks/WorkbooksMetadata.json:1423]: DataType should be the table name, so please change it to 
**AzureDiagnostics**

Thanks!!!

File: Workbooks/WorkbooksMetadata.json [hunk 7]
```diff
@@ -1415,5 +1415,18 @@
...
+      "dataConnectorsDependencies": ["AzurePurview"],
```
Comment (v-rucdu) [inline Workbooks/WorkbooksMetadata.json:1424]: DataConnectorDependencies should have the value of  **id** attribute from DataConnector json file.

**Review by v-rucdu — CHANGES_REQUESTED** [review]

File: Workbooks/WorkbooksMetadata.json (hunk 7 above)
Comment (libarson) [reply Workbooks/WorkbooksMetadata.json]: DataConnectorDependencies has been updated to have the id attribute for the data connector.

File: Workbooks/WorkbooksMetadata.json (hunk 6 above)
Comment (libarson) [reply Workbooks/WorkbooksMetadata.json]: DataTypesDepdendencies changed to AzureDiagnostics.

**Review by v-rucdu — APPROVED** [review]

**Review by v-jayakal — APPROVED** [review]

**Pull request merged**